main.go

```diff
@@ -182,6 +184,7 @@ func main() {
 	flag.IntVar(&opts.ListenPort, "listen-port", 4040, "HTTP port to listen on")
 	flag.StringVar(&opts.Format, "format", `$remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"`, "NGINX access log format")
 	flag.StringVar(&opts.Namespace, "namespace", "nginx", "namespace to use for metric names")
+	flag.StringVar(&opts.WebConfigFile, "web-config", "", "Web Configuration file")
```

Comment:
Martin, I feel that from user perspective it's important to have a unified way of working (ideally) across all prometheus exporters: same way of doing TLS configuration, same way of parametrizing things like listening port etc etc. When a team start using Prometheus they are not using only nginx-log epxorter or only oracle exporter. They usually have a bunch of exporters and also automated ways to install them. It's not helping them to have so different options for each exporter. This can only end like in the world of connectors (usb, hdmi etc): each producer has its own "best" standard and we the consumers are struggling with all kind of adaptors. When we talk about the exporter-toolkit I feel it worth taking the risk and do it in the same way (I'm not even sure why are they keep labeling it like that - maybe we should ask when are they confident enough to remove that if that's the issue here).